netshare/learn/utils/word2vec_embedding.py

```diff
@@ -35,4 +46,99 @@ def word2vec_train(
         model.save(model_path)
     logger.info(f"Word2Vec model is saved at {model_path}")
 
-    return model_path
+    return None
+
+
+def get_word2vec_type_col(word2vec_cols: List[Any]) -> Dict[str, List[str]]:
+    dict_type_cols: Dict[str, List[str]] = {}
+    for col in word2vec_cols:
+        type = col.encoding
+        if type not in dict_type_cols:
+            dict_type_cols[type] = []
+        dict_type_cols[type].append(col.column)
+    return dict_type_cols
```

Comment:
```suggestion
    dict_type_cols: Dict[str, List[str]] = defaultdict(list)
    for col in word2vec_cols:
        dict_type_cols[col.encoding].append(col.column)
    return dict_type_cols
```
`from collections import defaultdict` This is one of my best utilities :)